egs/hkust/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -7,31 +7,25 @@
 
```

Comment:
similarly as with the other scripts -- please mention the origin if you know it.